windows/ReactNativeWebView/ReactWebView2Manager.cpp

```diff
@@ -134,13 +140,13 @@ namespace winrt::ReactNativeWebView::implementation {
             }
         }
         else if (commandId == L"reload") {
-            webView.Refresh();
+            webView.Reload();
         }
         else if (commandId == L"stopLoading") {
-            webView.Stop();
```

Comment:
why is this not calling Stop anymore?

Reply:
I think for WV2 you'll want `webView.CoreWebView2().Stop()`

Reply:
That makes sense! I combed through the webview2 APIs and couldn't find a stop method. I'll try that option.